docs/core-features/fhe_basics.md

```diff
@@ -73,9 +71,8 @@ module {
   }
 }
 ```
+There are 2 things to keep in mind about PBS:
 
-The only thing you should keep in mind is that it adds a constraint on the input type, and that is the reason behind having a maximum bit-width supported in Concrete.
-
-Second takeaway is that PBS are the most costly operations in FHE, the less PBS in your circuit the faster it will run. It is an interesting metrics to optimize (you will see that Concrete could give you the number of PBS used in your circuit).
+- **Input type constraints**: PBS operations adds constraints on input typo and thus limits the maximum bit-width supported in Concrete.
+- **PBS performance impact**: PBS operations are costly, so minimizing the number of PBS can improve circuit performance. PBS cost also varies with input precision (for example, an 8-bit PBS is faster than a 16-bit PBS). To learn more about optimizing PBS, refer to the [Optimization](/docs/optimization/self.md) section.
```

Comment:
I wonder if this link [Optimization](/docs/optimization/self.md) works. It's strange to start with /

Reply:
I'll update it to use `../`